frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -404,7 +402,7 @@ function removeForestClientFromList(index: number) {
                                 :name="'verifyIdir'"
                                 :label="'Verify'"
                                 @click="
-                                    verifyIdentity(
+                                    verifyUserId(
```

Comment:
rename this method from verifyIdentity to verifyUserId, feel that's more clear about which form field we're trying to verify